tests/stubs/devices/services/temperature.go

```diff
@@ -0,0 +1,73 @@
+package services
+
+import (
+	"crypto/tls"
+	"fmt"
+	MQTT "github.com/eclipse/paho.mqtt.golang"
+	"github.com/paypal/gatt"
+	"log"
+	"time"
+)
+
+type Token interface {
+	Wait() bool
+	WaitTimeout(time.Duration) bool
+	Error() error
+}
+
+var TokenClient Token
+var ClientOpts *MQTT.ClientOptions
+var Client MQTT.Client
+var mqttServer string
+
+func HubClientInit(server, clientID, usename, password string) *MQTT.ClientOptions {
+	opts := MQTT.NewClientOptions().AddBroker(server).SetClientID(clientID).SetCleanSession(true)
+	if usename != "" {
+		opts.SetUsername(usename)
+		if password != "" {
+			opts.SetPassword(password)
+		}
+	}
+	tlsConfig := &tls.Config{InsecureSkipVerify: true, ClientAuth: tls.NoClientCert}
+	opts.SetTLSConfig(tlsConfig)
+	return opts
+}
+
+func MqttConnect() {
+	mqttServer = "tcp://<machineip where broker is running>:1883"
```

Comment:
<machineip where broker is running>  --> this needs to be replaced before running the mock server ?
can we have this config based ?

Reply:
Updated and removed mqtt code